Text/CustomNER/README.md

```diff
@@ -1,35 +1,81 @@
+    - [Index](#index)
+    
+    - [Indexer](#indexer)
+- [Automated Deployment](#automated-deployment)
+
+- [Testing](#testing)
+
 ---
 
-# Steps    
+# Steps
+
+#### Create or reuse a Custom NER project
+
+In order to use Custom NER, we need a language resource and a trained (and deployed) project to be used in recognizing the custom entities. If they aren't previously created, now is the time to do that. A good place to start is [Quickstart - Custom named entity recognition (NER) - Azure Cognitive Services | Microsoft Docs](https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/custom-named-entity-recognition/quickstart?pivots=language-studio).
+
+After this step you should have:
+
+* A Language resource.
+
+* A project
+
+* A deployment
 
-1. Create or reuse a Text Analytics resource. Creation can be done from the Azure portal or in [Language Studio](https://language.azure.com/home)
-2. Train your model with a dataset (a sample train and eval dataset can be found [here](https://github.com/Azure-Samples/cognitive-services-sample-data-files/tree/master/language-service/Custom%20NER/loan%20agreements) in case you dont have docs to work with) and deploy it. In case you are not familiar with Custom NER, this is a simple [tutorial](https://docs.microsoft.com/azure/cognitive-services/language-service/custom-named-entity-recognition/quickstart?pivots=language-studio#upload-sample-data-to-blob-container) to guide you
-3. Create a Python Function in Azure, for example this is a good [starting point](https://docs.microsoft.com/azure/azure-functions/create-first-function-vs-code-python)
-4. Clone this repository
-5. Open the folder in VS Code and deploy the function, find here a [tutorial](https://docs.microsoft.com/azure/search/cognitive-search-custom-skill-python)
-6. Fill your Functions appsettings with the custom details from your deployment ('TA_ENDPOINT', 'TA_KEY', 'DEPLOYMENT', 'PROJECT_NAME' with the info you got in Language Studio after you deployed the model
-7. Add a field in your index where you will dump the enriched entities, more info [here](#sample-index-field-definition)
-8. Add the skill to your skillset as [described below](#sample-skillset-integration)
-9. Add the output field mapping in your indexer as [seen in the sample](#sample-indexer-output-field-mapping)
-10. Run the indexer 
+#### Deploy the powerskill to Azure
 
-## Sample Input:
+A powerskill is basically just an Azure Function written to be used as a custom skill in an Azure Cognitive Search pipeline. To deploy a function, an Azure App resource is needed. Notice the differnce between an Azure Function and an Azure App. If one is not already available, now is the time to create one. A good article that goes through all the steps of creating a simple Azure Function in Python is [Create a Python function using Visual Studio Code - Azure Functions | Microsoft Docs](https://docs.microsoft.com/en-us/azure/azure-functions/create-first-function-vs-code-python).
```

Comment:
Typo: "differnce" -> "difference"

Reply:
Fixed in [a417a1b](https://github.com/Azure-Samples/azure-search-power-skills/pull/97/commits/a417a1b3c3c6dae7c6056b620329136fc4d4b12c)